tasks/install_psmdb_full.yml

```diff
@@ -27,6 +27,9 @@
       - percona-server-mongodb-shell
       - percona-server-mongodb-tools
       - percona-server-mongodb-debuginfo
+      - percona-server-mongodb-mongos-debuginfo
```

Comment:
I don't think `percona-server-mongodb-mongos-debuginfo`, `percona-server-mongodb-shell-debuginfo` and `percona-server-mongodb-tools-debuginfo` should be added here since this is for centos 6/7.